adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -0,0 +1,129 @@
+//
+//  GADPangleRTBBannerRenderer.m
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import "GADPangleRTBBannerRenderer.h"
+#import "GADMediationAdapterPangleConstants.h"
+#import <BUAdSDK/BUAdSDK.h>
+#import "GADMAdapterPangleUtils.h"
+#include <stdatomic.h>
+
+static CGSize const kPangleBannerAdSize320x50 = (CGSize){320,50};
+static CGSize const kPangleBannerAdSize300x250 = (CGSize){300,250};
+static CGSize const kPangleBannerAdSize728x90 = (CGSize){728,90};
```

Comment:
Prefer removing the `k` prefix to match Google style. So `kPangle...` -> `pangle...`.